docs/ARCHITECTURE.adoc

```diff
@@ -1,11 +1,11 @@
 == Application architecture
 
-The application is divided into two main parts: link:../src/main/java/io/jenkins/pluginhealth/scoring/probes/Probe.java[`Probe`] and link:../src/main/java/io/jenkins/pluginhealth/scoring/scores/Scoring.java[`Scoring`].
+The application is divided into two main parts: link:/core/src/main/java/io/jenkins/pluginhealth/scoring/probes/Probe.java[`Probe`] and link:/core/src/main/java/io/jenkins/pluginhealth/scoring/scores/Scoring.java[`Scoring`].
 
 === What is a `Probe`?
 
 Probes have the responsibility to fetch data for each plugins.
-They reach external system, like GitHub, the Jira, etc, and register the wanted data into a link:../src/main/java/io/jenkins/pluginhealth/scoring/model/ProbeResult.java[`ProbeResult`].
+They reach external system, like GitHub, the Jira, etc, and register the wanted data into a link:/core/src/main/java/io/jenkins/pluginhealth/scoring/model/ProbeResult.java[`ProbeResult`].
```

Comment:
This is not fixing the problem in the preview.
I think it needs to be `../core/`

Reply:
But it seems to work pretty fine in my forked repository.
Kindly have a look and lemme know If I still need to change it.
https://github.com/maulikpareek/plugin-health-scoring/tree/docs

Reply:
GitHub does not parse the `.adoc` correctly. If you test your syntax in the local it won't work.

Reply:
alright! fixed it with ../core